From 0ba6c63f629423702bc836ae1a740f72d71aee99 Mon Sep 17 00:00:00 2001
From: Thomas Huth <thuth@redhat.com>
Date: Wed, 1 Oct 2025 13:57:59 +0200
Subject: [PATCH] redhat: Fix -rhel7.6.0 machine type

JIRA: https://issues.redhat.com/browse/RHEL-105900

During the rebase to QEMU 10.1, we lost the S390_FEAT_LIST_QEMU_V3_1
setting for the -rhel7.6.0 machine type. Now that the upstream
patches that removed this setting got reverted, we can re-establish
it for the -rhel7.6.0 machine.

Signed-off-by: Thomas Huth <thuth@redhat.com>
---
 hw/s390x/s390-virtio-ccw.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/s390x/s390-virtio-ccw.c b/hw/s390x/s390-virtio-ccw.c
index 40d9c209f4..79fe544224 100644
--- a/hw/s390x/s390-virtio-ccw.c
+++ b/hw/s390x/s390-virtio-ccw.c
@@ -1365,7 +1365,7 @@ DEFINE_CCW_MACHINE(8, 2, 0);
 
 static void ccw_rhel_machine_7_6_0_instance_options(MachineState *machine)
 {
-    static const S390FeatInit qemu_cpu_feat = { };
+    static const S390FeatInit qemu_cpu_feat = { S390_FEAT_LIST_QEMU_V3_1 };
 
     ccw_rhel_machine_8_2_0_instance_options(machine);
 
